docs/contributing/how-to-label-an-issue.md

```diff
@@ -1,97 +1,99 @@
 ---
-title: How to label an issue
----
 
-## What are issue labels?
+title: किसी इशू को लेबल कैसे करें
+---
 
-Issue labels are a tool in GitHub that are used to group issues into one or more categories.
+## लेबल इशू क्या हैं?
 
-[Check out Gatsby's labels (and their descriptions)](https://github.com/gatsbyjs/gatsby/issues/labels)
+इशू लेबल GitHub में एक उपकरण है जो एक या अधिक कैटगोरी में ग्रुप के इशू के लिए उपयोग किया जाता है।
 
-## Why label issues?
+[Gatsby के लेबल देखें (और उनके विवरण)](https://github.com/gatsbyjs/gatsby/issues/labels)
 
-Gatsby is a very active project with many new issues opened each day. Labelling issues helps by identifying:
+## लेबल इशू क्यों?
 
-- good issues for new contributors to work on
-- reported and confirmed bugs
-- feature requests
-- duplicate issues
-- issues that are stalled or blocked
+Gatsby एक बहुत ही सक्रिय परियोजना है जिसमें प्रत्येक दिन कई नए इशू खोले जाते हैं। लेबलिंग इशू की पहचान करने में हेल्प करता है:
 
-## Who can label issues?
+- नए कन्ट्रिब्यटर के लिए अच्छे इशू पर काम करना
+- रिपोर्टेड और कनफर्म्ड bug
+- सुविधा का अनुरोध(रिक्वेस्ट)
+- डुप्लिकेट इशू
+- ऐसे इशू जो रुके हुए या अवरुद्ध(ब्लॉक्ड) हैं
 
-Anyone who's a member of the [Gatsby Maintainers team](https://github.com/orgs/gatsbyjs/teams/maintainers) can label issues.
+## इशू को कौन लेबल कर सकता है?
 
-You can get an invite to the team by having a Pull Request merged into the Gatsby project. Check out the list of [`help wanted`](https://github.com/gatsbyjs/gatsby/labels/%F0%9F%93%8D%20status%3A%20help%20wanted) issues and the [How To Contribute Guide](/contributing/how-to-contribute/) to get started.
+[`Gatsby मेंटेनर्स टीम`](https://github.com/orgs/gatsbyjs/teams/maintainers) का कोई भी सदस्य(मेंबर) हो वह इशू को लेबल कर सकता हैं।
 
-**NOTE:** If you’ve already had a pull request merged and you have _not_ been invited to the maintainers team, please go to [the dashboard](https://store.gatsbyjs.org/) and request a discount code. You should get an invite to the team — _and you get free Gatsby swag!_ If that doesn’t work, please email team@gatsbyjs.com and we'll get you invited.
+पुल रिक्वेस्ट को Gatsby प्रोजेक्ट में मर्ज कर आप टीम को इन्वाइट कर सकते हैं। लिस्ट चेक करे
+ [`हेल्प चाहिए`](https://github.com/gatsbyjs/gatsby/labels/%F0%9F%93%8D%20status%3A%20help%20wanted) इशू और [कैसे कंट्रीब्यूट करे
```

Comment:
```
लिस्ट चेक करे
 [`हेल्प चाहिए`](https://github.com/gatsbyjs/gatsby/labels/%F0%9F%93%8D%20status%3A%20help%20wanted) इशू और [कैसे कंट्रीब्यूट करे
गाइड](/contributing/how-to-contribute/) स्टार्ट करने के लिए।
```
=>
```
 स्टार्ट करने के लिए [`help wanted`](https://github.com/gatsbyjs/gatsby/labels/%F0%9F%93%8D%20status%3A%20help%20wanted) इश्यूज की लिस्ट और [कैसे कंट्रीब्यूट करे
गाइड](/contributing/how-to-contribute/) चेक करे।
```

Reply:
```
स्टार्ट करने के लिए [`help wanted`](https://github.com/gatsbyjs/gatsby/labels/%F0%9F%93%8D%20status%3A%20help%20wanted)  इश्यूज की लिस्ट और [कैसे कंट्रीब्यूट करे
गाइड](/contributing/how-to-contribute/) चेक करे।
```
=>
```
स्टार्ट करने के लिए [`help wanted`](https://github.com/gatsbyjs/gatsby/labels/%F0%9F%93%8D%20status%3A%20help%20wanted) इश्यूज की लिस्ट और [कैसे कंट्रीब्यूट करे गाइड](/contributing/how-to-contribute/) चेक करे।
```

there are couple of broken spaces and line breaks. can you copy paste the complete sentence